README.md

```diff
@@ -1,37 +1,164 @@
 # Executable Mocks
 
-**This is not an officially supported Google product.**
+A tool that mocks executables to make tests more effective.
 
-## Example
+## Table of contents
+- [Description](#Description)
+- [Structure](#Structure)
+- [Getting started](#Getting-started)
+- [Usage](#Usage)
+- [Authors](#Authors)
+- [License](#License)
+- [Disclaimer](#Disclaimer)
 
-As an example of how to use this tool, we are going to mock the `tar` utility.
+## Description
 
-[compress](https://github.com/googleinterns/executable-mocks/examples/tar/compress.sh) is a program that uses the `tar` utility. The program echoes some lines and stores some files in a new file.
+Testing a program, which consists of multiple utilities, sometimes can be resource-intensive,
+as the build utilities that we use in tests are very slow. Since every utility is already tested, and we only need 
```

Comment:
Since every utility is already tested, ___we only need to ...

Reply:
I modified it a little bit.

Reply:
great